app/lib/actions/katello/repository/filtered_index_content.rb

```diff
@@ -17,7 +17,7 @@ def run
             repo.index_db_puppet_modules
           elsif repo.docker?
             # have to call Repository#index_db_docker_images to get the repo's tags
```

Comment:
I am not sure if this works 100% the same since tags are units now instead of repo metadata